runtime/src/constants.rs

```diff
@@ -5,6 +5,8 @@ pub mod currency {
 	pub const DOLLARS: Balance = HDX * 10;	// 10 HDX ~= 1 $
 	pub const CENTS: Balance = DOLLARS / 100;
 	pub const MILLICENTS: Balance = CENTS / 1_000;
```

Comment:
just fyi: Gav mentioned that he considers the naming choice of "Dollar" a mistake now (and I agree with him ;-) )

Reply:
It was confusing at first but I guess it makes sense. We want to have this dynamic later on. The first token that will be in the pool is DAI (dollar), we'd like to make it in a way that fees paid will reflect the change in price of the token automatically so that validators are not under or overpaid.

Reply:
Do you have any better suggestion for now? :)

Reply:
even DAI would be better

Reply:
but the exchange ratio between DAI and HDX will be variable
I would go with semantic names like "existential deposit" or "minimum identity deposit" etc.
of course you can hard code a rough guess at the DAI <-> HDX conversion if that seems sensible, you will just (probably) have to update this frequently if the value changes a lot

Reply:
Yeah, right now it's fixed, so It's hardcoded as the price in DAI for HDX currently is ~0.08059$ so we rounded it to 10 cents. It won't change until we make HDX trade-able. This should go away completely when we implement dynamic prices (like existential deposit). We'd still probably want to make it pegged to hardware costs unless everything can be paid in crypto :)